com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs.hlsl

```diff
@@ -39,7 +39,26 @@
 #define FULLSCREENDEBUGMODE_MAX_MATERIAL_FULL_SCREEN_DEBUG (29)
 #define FULLSCREENDEBUGMODE_SCREEN_SPACE_REFLECTIONS_PREV (30)
 #define FULLSCREENDEBUGMODE_SCREEN_SPACE_REFLECTIONS_ACCUM (31)
-#define FULLSCREENDEBUGMODE_WORLD_SPACE_POSITION (32)
+#define FULLSCREENDEBUGMODE_VOLUMETRIC_CLOUDS (15)
+#define FULLSCREENDEBUGMODE_VOLUMETRIC_CLOUDS_SHADOW (16)
```

Comment:
something is wrong here in code generation

Reply:
Bad merge here